Entities/MOSRotating.h

```diff
@@ -848,6 +848,18 @@ ClassInfoGetters
 
 	void SetTravelImpulse(Vector impulse) { m_TravelImpulse = impulse; }
 
+	/// <summary>
+	/// Gets this MOSRotating's Gib Sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this MOSRotating's Gib Sound.</returns>
+	SoundContainer* GetGibSound() const { return m_GibSound; }
+
+	/// <summary>
+	/// Sets this MOSRotating's Gib Sound.
+	/// </summary>
+	/// <param name="newSound">The new SoundContainer for this MOSRotating's Gib Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.